golem-component-service-base/tests/db/migration/sqlite/001__init.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE components
+(
+    namespace           text    NOT NULL,
+    component_id        uuid    NOT NULL,
+    name                text    NOT NULL,
```

Comment:
Please consider adding a unique index on `(name, namespace)` - in this order. To bi able to use it to search by `name` only.